experimental/sev_guest/src/msr.rs

```diff
@@ -257,30 +258,41 @@ pub struct RegisterGhcbGpaResponse {
     ghcb_gpa: u64,
 }
 
+#[derive(Debug, Snafu)]
+pub enum RegisterGhcbGpaError {
```

Comment:
How did we decide on these errors and is there some documentation on when those can happen?

Reply:
I'll let Conrad elaborate; I determined these errors by looking at code and giving enum entries for all different strings :)